examples/network-explorer/server/server.ts

```diff
@@ -67,148 +51,11 @@ const canvasApp = await Canvas.initialize({
-	} else if (req.query.type === "action" || req.query.type === "session") {
-		messageIndexEntries = await queries.selectMessages(before, req.query.type, numMessagesToReturn)
-	} else {
-		res.status(StatusCodes.BAD_REQUEST)
-		res.end()
-		return
-	}
-
-	const result = []
-	for (const messageIndexEntry of messageIndexEntries.rows) {
-		const signedMessage = await canvasApp.getMessage(messageIndexEntry.id)
-		// during initialization, the app may be missing messages, and
-		// we shouldn't send null signature/message values to the client
-		if (signedMessage !== null) {
-			result.push({
-				id: signedMessage.id,
-				signature: signedMessage.signature,
-				message: signedMessage.message,
-				branch: signedMessage.branch,
-			})
-		}
-	}
-
-	res.status(StatusCodes.OK)
-	res.setHeader("content-type", "application/json")
-	res.end(json.encode(result))
-})
-
-expressApp.get("/index_api/counts", async (req, res) => {
-	const countsQueryResult = (await queries.selectCounts()).rows
-
-	let actionCount: number = 0
-	let sessionCount: number = 0
-
-	for (const row of countsQueryResult) {
-		if (row.type === "action") actionCount = row.count
-		if (row.type === "session") sessionCount = row.count
-	}
-
-	const addressCountResult = (await queries.selectAddressCount()).rows
-	const addressCount = parseInt(addressCountResult[0].count, 10)
-
-	// TODO: get these fields from the canvas app object
-	// const connectionCount = canvasApp.libp2p.getConnections().length
-	// const connections = (connectionsMap[row.topic] = canvasApps[row.topic]?.libp2p
-	// 	.getConnections()
-	// 	.map((c) => c.remoteAddr.toString())
-	// 	.join(", "))
-
-	const connectionCount = 0
-	const connections = "-"
-
-	const result = {
-		topic,
-		address_count: addressCount,
-		connection_count: connectionCount,
-		connections,
-		action_count: actionCount,
-		session_count: sessionCount,
-	}
-
-	res.json(result)
-})
-
-expressApp.get("/index_api/latest_session/", async (req, res) => {
-	if (
-		!req.query.did ||
-		typeof req.query.did !== "string" ||
-		!req.query.public_key ||
-		typeof req.query.public_key !== "string"
-	) {
-		res.status(StatusCodes.BAD_REQUEST)
-		res.end()
-		return
-	}
-
-	const sessionId = await canvasApp.getSession({
-		did: req.query.did,
-		publicKey: req.query.public_key,
-	})
-
-	if (!sessionId) {
-		res.status(StatusCodes.NOT_FOUND)
-		res.end()
-		return
-	}
-
-	const signedMessage = await canvasApp.getMessage(sessionId)
-	if (signedMessage === null || signedMessage.message.payload.type !== "session") {
-		res.status(StatusCodes.NOT_FOUND)
-		res.end()
-		return
-	}
-
-	// return using ipld json stringify
-	res.status(StatusCodes.OK)
-	res.setHeader("content-type", "application/json")
-	res.end(json.encode(signedMessage.message.payload))
-})
+expressApp.use("/api/", canvasApiApp)
 
 expressApp.listen(HTTP_PORT, HTTP_ADDR, () => {
 	console.log(`> Ready on http://${HTTP_ADDR}:${HTTP_PORT}`)
```

Comment:
I don't think we need the "server" part of network explorer anymore - we should just be able to point the network explorer client at any Canvas node that has APIs exposed.